nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -2539,6 +2539,12 @@ private ProcessorStatus getProcessorStatus(final RepositoryStatusReport report,
         return status;
     }
 
+    private boolean isValid(ProcessorNode procNode) {
+        try (final NarCloseable narCloseable = NarCloseable.withNarLoader()) {
+            return procNode.isValid();
```

Comment:
I believe we want to wrap the call into validate() on the Processor in procNode.isValid() rather than the entire isValid() method [1].

[1] https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/StandardProcessorNode.java#L911